.github/workflows/coverage.yml

```diff
@@ -14,14 +14,14 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v2
       with:
-        python-version: pypy3
+        python-version: 3.9
```

Comment:
Hi! Sorry for the delay on this one. I'll get you a full review soon, but this change (moving coverage to run on 3.9) is another good one to extract into its own tiny PR please! Appreciated.

Reply:
Moved to new PR https://github.com/Julian/jsonschema/pull/823

Also I cleaned the commits for the fix that has been applied with https://github.com/Julian/jsonschema/pull/820

Reply:
Fantastic thanks! I'll have a look there too.